src/routes/users/fetchAllUsers.ts

```diff
@@ -50,7 +50,7 @@ export const schema = {
     200: {
       title: 'Users',
       type: 'array',
-      propiesties: UserSchema,
+      properties: UserSchema,
```

Comment:
Now that we have the swagger UI configured, please validate that this endpoint is displayed correctly.